yb-voyager/cmd/exportSchema.go

```diff
@@ -132,3 +134,17 @@ func clearSchemaIsExported() {
 		utils.ErrExit("clear schema is exported: update migration status record: %s", err)
 	}
 }
+
+func updateIndexesInfoInMetaDB() {
+	log.Infof("updating indexes info in meta db\n")
+	indexesInfo := source.DB().GetIndexesInfo()
+	err := metadb.UpdateJsonObjectInMetaDB(metaDB, metadb.INDEXES_INFO_KEY, func(record *[]utils.IndexInfo) {
+		// updating if indexesInfo is not nil
+		if indexesInfo != nil {
```

Comment:
this is the only and first time you are updating this json object, right? 
so, why would this not be nil?
Also, what if it is nil?

Reply:
yes you are right.
The issue is we can't append a nil to this slice in golang, it errors out.